wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -3552,15 +3644,20 @@ WOLFSSL_LOCAL int tsip_SignRsaPkcs(wc_CryptoInfo* info, TsipUserCtx* tuc)
 
     WOLFSSL_ENTER("tsip_SignRsaPkcs");
 
-    if (info == NULL || tuc == NULL || tuc->ssl == NULL) {
+    if (info == NULL || tuc == NULL 
+    #ifndef WOLFSSL_RENESAS_TSIP_CRYPTONLY
+    || tuc->ssl == NULL
+    #endif
+    ) {
             ret = BAD_FUNC_ARG;
     }
-
+    
+#ifndef WOLFSSL_RENESAS_TSIP_CRYPTONLY
```

Comment:
“#ifdef WOLFSSL_RENESAS_TSIP_TLS” seems more correct here. Does it fit your intentions?

Reply:
Thanks, Fixed